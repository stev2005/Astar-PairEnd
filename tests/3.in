GGACCGGATCAGGTCTCGAATCTTATGACAATTAAAGTACAAATTCGATGCGTACTTATAAATTAAATTAATCAGACACATGGGGAGGACGACGAGGACCACACGTGTTTAGCGCTGTCGAATGGTAATCCTGCCAATAAGAGTGTGCGATTAG
15
GCCGAGTAAACCCACCGCTCTTTACTAATGCCAGTAAGACTCGACTCTAAATTAACTACTGTAGCATGGTTCCCCGAGGATGTGGTTGTCCGA
ACCTTATGCATTGTCATATTTCGGTGTAATGACTAATTTCTGACGCGCGCACGTTGCTGCG
CTACATATATCACTGTCAACCGAA
CTTGGTACTCCTGATCGAGCGCGGCATAATAGTTACTAGGGACAACCGCAATCGCTGATTTTCGTCTGGGAAGGAGTCCACCTTTAGGACCTAGGTTCCCTGCC
CATTCTAGGTCG
TGGACTGTGGCATAGCGGGGCCTAGCGCGATGGCGTACTTTATGACTGT
AATTACAGTTGGCATTTAAGTTGGGTTCCGTGGTCGAGATCGCGGCCCCGAACGGTCCCTTACCTGTAATTCCCTTGCATTATATCAAGCTCCAGCTCCTAACCCAATCAGACC
A
CAAGACTATATACTAGACTACTGGCGTTTTCATGGTTGATGTTAGAGGCCTTACCGTACTTGTGAGGAAGTGCGGAGAGAGCTGGAAGACCTAACAGT
GCTCTGTTAACAGTTACTCTTGAGGCATTTTGCTTACGTCGAGA
CCCCGAAACTTGTGCGCTTAGTCCGTTTCAGGCTTTAATCTGTGAAAGAAGGTATGTTCATTGATGT
GGACCTTCTTTTTGGTGCCCAGCTCTTACGTTCTAGGTTGTTCGCTCTCGA
ACCGAAGGTGCACGGAC
GAATTCTAAATACTCGCGTAAGGCGTTATCCCCGCATACAAAACTGACATCACTCTGCTGGATTGAAGACGACGGAAGCACGAGCGCATAGCACATG
CGCTTGCCTGTAGATTGACTAATG
