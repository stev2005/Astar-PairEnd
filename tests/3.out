44
28
10
52
4
22
56
0
45
20
31
24
6
42
9
